packages/xo-web/src/xo-app/home/index.js

```diff
@@ -1033,6 +1048,28 @@ export default class Home extends Component {
               </div>
             ) : (
               <div>
+                {type === 'VM' && (
+                  <OverlayTrigger
+                    trigger='click'
+                    rootClose
+                    placement='bottom'
+                    overlay={
+                      <Popover className={styles.selectObject} id='vmPopover'>
+                        <Select
+                          autoFocus
+                          onChange={this.linkState('vmsFilter')}
```

Comment:
you can use `simpleValue` property if you only want to use the option value.

Reply:
Done.